packages/back-end/src/util/mongo.util.ts

```diff
@@ -157,3 +161,40 @@ export function removeMongooseFields<T>(
 export function getCollection(name: string) {
   return mongoose.connection.db.collection(name);
 }
+
+type StoredLimits = {
+  requests: number | "unlimited";
+  bandwidth: number | "unlimited";
+};
+
+const backendLimits = ({
+  requests: cdnRequests,
+  bandwidth: cdnBandwidth,
+}: StoredLimits) => ({ cdnRequests, cdnBandwidth });
+
+export const getOrgUsageLimits = async (
+  organization: string
+): Promise<UsageLimits> => {
+  const mongodb = new MongoClient(MONGODB_URI);
```

Comment:
We can't reuse Mongoose but we could make this a singleton: 
```
let cachedClient: MongoClient | null = null;
let cachedDb: Db | null = null;

export async function getMongoUsageDb(): Promise<Db> {
  if (!cachedClient) {
    cachedClient = new MongoClient(MONGODB_URI);
    await cachedClient.connect();
    cachedDb = cachedClient.db("usage"); // Pre-select the "usage" database
  }
  return cachedDb!;
}

export async function closeMongoUsageDb(): Promise<void> {
  if (cachedClient) {
    await cachedClient.close();
    cachedClient = null;
    cachedDb = null;
  }
}
```

Then we should call `closeMongoClient` in server.ts in SIGINT and SIGTERM for correctness sake.